### DISCUSSION TITLE
6N137 instead of 6N138?

### DISCUSSION THREAD

**Discussion by souravlal** [marcel-licence/esp32_midi_sampler#27] [Q&A] [answered]
Can I use 6N137 instead of 6N138?
Only 6N137 available in my region.

**Comment by marcel-licence** [comment, marked as answer]
This may work. I am currently using H11L1 instead. You should be able to find schematics using another opto coupler. Just keep in mind that ESP32 inputs may be not 5V tolerant so if you connect a pull-up resistor it should be pull to 3.3V instead of 5V.

**Reply by souravlal** [reply]
Is there a 3.3V output pin on esp32 audio kit to power the optocoupler?
How do you done this?
